integration-tests/tests/run-workflow.js

```diff
@@ -58,22 +59,27 @@ const launchWorkflowAndWaitForSuccess = async page => {
   ])
   // stopLoggingPageAjaxResponses()
 
-  const disablePageLogging = logPageConsoleMessages(page)
   const start = Date.now()
   await pRetry(async () => {
     try {
       await findInGrid(page, 'Succeeded', { timeout: 65 * 1000 }) // long enough for the submission details to refresh
     } catch (e) {
       try {
         await findInGrid(page, 'Running', { timeout: 1000 })
-        console.info(`Workflow is running, elapsed time (minutes): ${(Date.now() - start) / (1000 * 60)}`)
+        rawConsole.info(`Workflow is running, elapsed time (minutes): ${(Date.now() - start) / (1000 * 60)}`)
       } catch (e) {
-        console.info(`Workflow not yet running, elapsed time (minutes): ${(Date.now() - start) / (1000 * 60)}`)
+        rawConsole.info(`Workflow not yet running, elapsed time (minutes): ${(Date.now() - start) / (1000 * 60)}`)
       }
       throw new Error(e)
     }
-  }, { retries: 15, factor: 1 })
-  disablePageLogging()
+  },
+  // Note about retries value:
+  // If there is a large backlog of workflows in the environment (for example, if there has been a large submission), workflows
+  // might sit in "Submitted" for a very long time. In that situation, it is possible that this test will fail even with a 12-minute
+  // retries value. The test runner will kill any test that goes over 15 minutes though, so keep retries under that threshold.
+  // BW-1057 should further reduce this retries value after changes have been implemented to move workflows through Cromwell more quickly.
+  { retries: 12, factor: 1 }
```

Comment:
I had increased this from 5 retries to 15. But then when performance testing was going on in dev, we were seeing test failures because this test went over 15 minutes. There were log messages from this loop up until 13.x minutes, so reducing to 12 so that the cause of failure is more obvious (Workflow does not get to Succeeded in allotted time, vs. test runner killing the test for being too long).